x/executionlayer/client/util/util.go

```diff
@@ -108,3 +93,73 @@ func GetLocalWalletInfo(valueFromFromFlag string, kb keys.Keybase, cdc *codec.Co
 
 	return infoList[0], nil
 }
+
+func UnitConverterRemovePoint(src string) (string, error) {
```

Comment:
How about renaming them like below?
eg:
```go
type U512 = string
type HDAC = U512
type BIGSUN = U512

func ToBigsun(from HDAC) (BIGSUN, error) {}
func ToHDAC(from BIGSUN) (HDAC, error) {}
```

Reply:
If you have some points of discussion, but it's not critical, how about apply this first and do discuss and make new pr?

Reply:
https://github.com/hdac-io/friday/pull/86/commits/378ce2e23b5a163d451af7014129dc8952bde533

If we are using U512, I think there will be some confusion on how to make a rest.